### ISSUE TITLE
Forbid extra fields in config

### ISSUE THREAD

**Issue by stefsmeets** [CarbonCollective/fusion-dUQtools#102]
To prevent typo's, i.e. `subdirectory` instead of `subdir` (which has a default value, so it won't get flagged as missing), we can use `Extra.forbid`: https://pydantic-docs.helpmanual.io/usage/model_config/ 

```python
from pydantic import BaseModel, Extra

class Model(BaseModel, extra=Extra.forbid):
    ...
```

**Comment by stefsmeets** [comment]
Change behaviour globally:

https://pydantic-docs.helpmanual.io/usage/model_config/#change-behaviour-globally

**Comment by stefsmeets** [comment]
Closed by #103
